tools/prerelease-tool/main.go

```diff
@@ -0,0 +1,134 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"sort"
+	"time"
+
+	"github.com/google/go-github/github"
+	"golang.org/x/oauth2"
+)
+
+type prereleaseTool struct {
+	Data        []*github.RepositoryRelease
```

Comment:
Can we call this `Releases`?

Reply:
yes, I had already done that believe it or not during final cleanup, but obviously lost that change when I rebased from master :)